packages/vscode-extension/src/error/common.ts

```diff
@@ -58,7 +81,14 @@ export async function showError(e: UserError | SystemError) {
     VsCodeLogInstance.error(`code:${errorCode}, message: ${e.message}\n Help link: ${e.helpLink}`);
     // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
     VsCodeLogInstance.debug(`Call stack: ${e.stack || e.innerError?.stack || ""}`);
-    const buttons = recommendTestTool ? [runTestTool, help] : [help];
+    const buttons = recommendTestTool
+      ? shouldRecommendTeamsAgent
+        ? []
+        : [runTestTool, help]
+      : [help];
+    if (shouldRecommendTeamsAgent) {
+      buttons.push(troubleshootErrorWithTeamsAgentButton);
```

Comment:
Can this line90 be merged into line 86? And just to confirm that this means replacing test tool recommendation with troubleshooting button.

Reply:
oh sorry. I just need line 89 and 90. Will remove line 84-88